demo/admin/src/documents/pages/blocks/TeaserItemBlock.tsx

```diff
@@ -27,6 +34,17 @@ export const TeaserItemBlock = createCompositeBlock(
                     label: <FormattedMessage id="teaserItemBlock.title" defaultMessage="Title" />,
                 }),
             },
+            titleHtmlTag: {
+                block: createCompositeBlockSelectField<TeaserItemBlockData["titleHtmlTag"]>({
+                    label: <FormattedMessage id="teaserItemBlock.titleHtmlTag" defaultMessage="Title HTML tag" />,
+                    defaultValue: "h3",
+                    options: ([1, 2, 3, 4, 5, 6] as const).map((number) => ({
+                        value: `h${number}`,
+                        label: <FormattedMessage id={`teaserItemBlock.headline${number}`} defaultMessage={`Headline ${number}`} />,
```

Comment:
Please do the same as here: https://github.com/vivid-planet/comet/pull/4379#discussion_r2313651646